tests/sysversioninfo.pyi

```diff
@@ -12,7 +12,7 @@ if sys.version_info[0j] == 2:  # Y003 Unrecognized sys.version_info check
 if sys.version_info[0] == (2, 7):  # Y003 Unrecognized sys.version_info check
     ...
 
-if sys.version_info[0] == '2':  # Y003 Unrecognized sys.version_info check
+if sys.version_info[0] == b'2':  # Y003 Unrecognized sys.version_info check
```

Comment:
Do these tests need to be changed? We are skipping the conditions of `if` statements.

Reply:
Yes, I think I changed this before I decided to just skip `if` completely. I'll change it back.